Tools/CISupport/Shared/de-duper

```diff
@@ -114,21 +119,28 @@ def main(args):
             if os.stat(full_path).st_nlink > 1:
                 continue
             sha_value = sha1(full_path)
+            sha_path = os.path.join(args.storage, f'{sha_value}.png')
+            if sha_path == full_path:
+                continue
```

Comment:
This should be unreachable with the check at the top of the `os.walk` loop, right? Can we just make this an assert instead, given this should never happen?